beamer/util.py

```diff
@@ -153,7 +153,7 @@ class _TokenData:
 }
 
 
-class TokenMatchChecker:
+class TokenChecker:
```

Comment:
You deserve a beer just for this change! It's definitely a better name now.

Reply:
🍺